bedrock/teach/templates/teach/smarton/tracking.html

```diff
@@ -0,0 +1,376 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "teach/smarton/base.html" %}
+
+{% block page_title %}{{ _('Get smart on tracking') }}{% endblock %}
+{% block page_desc %}{{ _('What you do online is your business, and you can keep it that way. Get smart on tracking.') }}{% endblock %}
```

Comment:
String added 20 October: "What you do online is your business, and you can keep it that way. Get smart on tracking."